presto-orc/src/main/java/com/facebook/presto/orc/reader/MapFlatSelectiveStreamReader.java

```diff
@@ -165,10 +165,12 @@ public MapFlatSelectiveStreamReader(
         }
         else {
             switch (keyOrcTypeKind) {
+                case BOOLEAN:
```

Comment:
unrelated change? Should it be a separate commit? What is this for?

Reply:
will move to a separate commit before the test